gds/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ if (NOT APPLE)
     set(CMAKE_INSTALL_RPATH $ORIGIN)
 endif ()
 
+# Find CUDA Toolkit for cudart and cufile.
+find_package(CUDAToolkit REQUIRED)
```

Comment:
To ensure cuFile is picked up from the CTK, we add this `find_package` line

Note that in the case where nvJPEG is used, we already had an equivalent line. Hence no change there